packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -106,7 +106,9 @@
     "editBoxCancelButton": "Cancel",
     "editBoxSubmitButton": "Submit",
     "messageReadCount": "Read by {messageReadByCount} of {remoteParticipantsCount}",
-    "actionMenuMoreOptions": "More Options"
+    "actionMenuMoreOptions": "More Options",
+    "messageContentDescription": "Message sent by {displayName} at {timeStamp}. {displayName} says {content}",
+    "myself": "myself"
```

Comment:
Could this key be `localSenderAriaLabel`. I'm conflicted whether it's better to announce "message sent by myself" or "message sent by you". In the UI, we show "(you)" in the participant list for the local user.
![image](https://user-images.githubusercontent.com/82062616/164838337-48cb293e-7cec-4588-85b3-71d4fd2c4765.png)

so I'm leaning towards "message sent by you" for the announcer as well.

In both cases (myself vs you), the key `localSenderArialLabel` sounds sensible, which validates that key name.

Reply:
I also learn towards 'message set by you' since I don't think we want the narrator talking in the first person for the user.

Reply:
Updated :)